scripts/globals/weaponskills.lua

```diff
@@ -370,7 +370,7 @@ local function getSingleHitDamage(attacker, target, dmg, ftp, wsParams, calcPara
     finaldmg = (dmg + consumeManaBonus(attacker)) * ftp * calcParams.pdif
 
     -- handle blocking
-    finaldmg = xi.combat.physical.handleBlock(target, attacker, finaldmg)
+    finaldmg, _ = xi.combat.physical.handleBlock(target, attacker, finaldmg)
```

Comment:
Yeah, this is signalling that the double return is probably not the best layout for this

Reply:
> Yeah, this is signalling that the double return is probably not the best layout for this

Yep, I agree, I have updated the PR as per my reply to the comment above and the updated summary of what the PR does. Thanks for your suggestion